src/app/left-menu/main/left-menu.component.ts

```diff
@@ -53,22 +52,54 @@ export class LeftMenuComponent implements OnDestroy, OnInit {
     }
 
     menuSearch() {
-        const keyword = this.searchKeyword.toLowerCase();
-        this.filteredMenuData = this.menuData.filter((value) => {
-            const name = value.name.toLowerCase();
-            const searchString = name + ', ' + value.equivalents;
-            return searchString.toLowerCase().indexOf(keyword) !== -1;
-        });
+        const keyword = this.getRealName(this.searchKeyword.toLowerCase());
+
+        const addFilteredMenuItem = (item: LeftMenuItem, itemsArray: LeftMenuItem[]): void => {
+            if(!item.items) return;
+
+            const filteredItems = item.items.filter((metaItem) => {
+                const itemName = this.getRealName(metaItem.Name).toLowerCase();
```

Comment:
Hm. I have bad news. We should keep in mind that word 'grid' must show all DataGrid group. In other words, we should make search by groups too.

Reply:
Should we make search by subgroups like "default/normal type" in "Button" group? Or only menu groups?

Reply:
And what about hightlighting for them?

Reply:
I think we should start with groups without subgroups. Highlight will be useful